server/utils/next_url.ts

```diff
@@ -25,8 +25,20 @@ export function composeNextUrlQeuryParam(request: KibanaRequest, basePath: strin
   return stringify({ nextUrl });
 }
 
+export const INVALID_NEXT_URL_PARAMETER_MESSAGE = 'Invalid nextUrl parameter.';
+
+/**
+ * Ensures the nextUrl parameter is a relative url, the nextUrl parameter should:
+ *   1. starts with '/'
+ *   2. not start with '//'
+ *   3. does not contain '@' in the path
+ * @param url
```

Comment:
Please add a description for return value

Reply:
my bad, this is commented for local testing